src/main/java/org/sonarqube/gradle/SonarResolverTask.java

```diff
@@ -133,4 +139,11 @@ void run() throws IOException {
     }
   }
 
+  // On gradle 9 projects with configuration cache enabled, we CANNOT rely on the project at execution time
+  private boolean canRelyOnProjectAtExecutionTime() {
+    boolean isConfigurationCacheDisabled = !getBuildFeatures().getConfigurationCache().getActive().get();
```

Comment:
The `.get()` documentation mentions that it can throw if there is no value. Should we err on the side of safety and add a default value?
```suggestion
  private boolean canRelyOnProjectAtExecutionTime() {
    boolean isConfigurationCacheDisabled = !getBuildFeatures().getConfigurationCache().getActive().getOrElse(true);
```